src/integTest/resources/server-config/src/main/liberty/config/server.xml

```diff
@@ -18,7 +18,7 @@
     </featureManager>
 
 
-    <httpEndpoint httpPort="${default.http.port}" httpsPort="${default.https.port}"
+    <httpEndpoint httpPort="9081" httpsPort="${default.https.port}"
```

Comment:
Maybe make this change more obvious so future developer have a better understand as to why the change is here and what the change tests.